plugins/ompgsql/ompgsql.c

```diff
@@ -363,8 +363,8 @@ setInstParamDefaults(instanceData *pData)
 	pData->trans_commit  = 100;
 	pData->trans_age     = 60;
 	pData->port          = 5432;
-	strncpy(pData->user, "postgres", strlen("postgres"));
-	strncpy(pData->pass, "postgres", strlen("postgres"));
+	strncpy(pData->user, "postgres", strlen("postgres") + 1);
```

Comment:
thinking twice... shouldn't that be ```..., sizeof(pData->user));``` as ```strncpy()``` intends to guard that the buffer is not overrun?

Reply:
That makes sense.